lib/src/platform_client.dart

```diff
@@ -29,12 +29,14 @@ class PlatformClient {
 
   final PlatformClientConfig _config;
   Session? _session;
-  MessagingClient? messagingClient;
```

Comment:
This should never have been writable. In fact, I don't think it should be accessible from `PlatformClient`, since, from an app's perspective, messaging is only possible in a `Room`. Something for a future refactor...